msteams-platform/m365-apps/extend-m365-meeting-app.md

```diff
@@ -0,0 +1,198 @@
+
+1. Sign in to [Microsoft Azure portal](https://portal.azure.com) with your test tenant account.
+1. Open the **App registrations** blade.
+1. Select the name of your meeting app to open its app registration.
+1. Select  **Expose an API** (under *Manage*).
+
+    :::image type="content" source="images/azure-app-registration-clients.png" alt-text="Screenshot shows the Authorized client Ids from the App registrations blade on Azure portal.":::
+
+1. In the **Authorized client applications** section, add following `Client Id` value:
+
+    |Microsoft 365 client application | Client ID |
+    |--|--|
+    |Outlook desktop | d3590ed6-52b3-4102-aeff-aad2292ab01c |
+
+For a full list of client application IDs used to extend Teams apps across the Microsoft 365 ecosystem, see [Teams tabs in Microsoft 365 and Outlook](extend-m365-teams-personal-tab.md#update-azure-ad-app-registration-for-sso).
+
+### Sideload your app using TeamsFx CLI
+
+The final step to running your app in Microsoft 365 and Outlook is to sideload your  [app package](..//concepts/build-and-test/apps-package.md) using the TeamsFx CLI tool.
+
+1. Package your Teams application ([manifest](../resources/schema/manifest-schema.md) and [app icons](/microsoftteams/platform/resources/schema/manifest-schema#icons)) in a zip file. If you used Teams Toolkit to create your app, you can easily do this using the **Zip Teams App Package** option in the **UTILITY** section of Teams Toolkit. Select the `manifest.json` file for your app and the appropriate environment (*local* or *dev*). Once created, Teams Toolkit will display a dialog linking to the zip file location.
+
+    :::image type="content" source="images/toolkit-zip-teams-app-package.png" alt-text="Screenshot shows the Zip Teams App Package option in Teams Toolkit extension for Visual Studio Code.":::
+
+1. The preview for Teams meeting app support in Outlook requires you to sideload your app package using TeamsFx CLI. (Sideloading from the *Manage your apps* option in Teams client is not currently supported.) Ensure TeamsFx CLI is installed:
+
+    ```bash
+    npm install -g @microsoft/teamsfx-cli
+    ```
+
+1. Use the following command to sideload your app package to Teams, replacing *<path\to\appPackage.zip>* with the path to your app package.
+
+    ```bash
+    teamsfx m365 sideloading --file-path <path\to\appPackage.zip>
+    ```
+
+After it's sideloaded to Teams, your meeting app will be available for use in both Teams and Outlook. It may take several minutes and a restart of the Outlook for Windows desktop client for your app to appear in the **Apps** menu.
+
+## Preview your meeting app in Outlook
+
+Here's how to preview your meeting app running on Outlook for Windows desktop:
+
+1. With your meeting app running (either remotely from Azure, or locally from Teams Toolkit `F5`), open Outlook for Windows desktop signed in to your test tenant account. If Outlook is already running, close and restart it again.
```

Comment:
```suggestion
1. Open Outlook for Windows with the meeting app running either remotely from Azure, or locally from Teams Toolkit. If Outlook is already running, close and restart again.
```

Reply:
I don't think we can reorder the clauses here--the meeting app needs to be running first before opening Outlook.